third_party/terraform/website/docs/r/sql_database_instance.html.markdown

```diff
@@ -240,6 +240,9 @@ The required `settings` block supports:
     for information on how to upgrade to Second Generation instances.
     A list of Google App Engine (GAE) project names that are allowed to access this instance.
 
+* `delete_protection` - (Optional, Default: `false`) Whether or not to allow Terraform to destroy the instance. Unless this field is set to false
```

Comment:
Please add additional documentation at the top of this page, similar to https://github.com/GoogleCloudPlatform/magic-modules/pull/3450/files#diff-cabec47f1c3393e21fdded661c4b35a8. The part I think is the most important to call out explicitly is that the field has to be false **in state**, which means setting it and then running `terraform apply` so that it sticks.

Also, the field name is `deletion_protection`.

Reply:
fixed